Mathlib/Tactic/Positivity/Basic.lean

```diff
@@ -146,3 +167,24 @@ def evalPow : PositivityExt where eval {u α} zα pα e := do
     | .nonnegative pa => ofNonneg pa (← synthInstanceQ (_ : Q(Type u)))
     | .nonzero pa => ofNonzero pa (← synthInstanceQ (_ : Q(Type u)))
     | .none => pure .none
+
+/-- Port note: in mathlib3 this alias is `private`, but mathlib4's `alias` does not
+(yet) support that. -/
+alias abs_pos ↔ _ abs_pos_of_ne_zero
```

Comment:
Mathlib3's `alias` doesn't support private either. I think the statement should just be written out.

Reply:
Done -- rewritten as a `private theroem`.